lib/tbot/config/service_spiffe_workload_api.go

```diff
@@ -92,6 +119,9 @@ type SPIFFEWorkloadAPIService struct {
 	// SVIDs is the list of SVIDs that the SPIFFE Workload API server should
 	// provide.
 	SVIDs []SVIDRequestWithRules `yaml:"svids"`
+	// Attestor is the configuration for the workload attestation process.
+	// TODO(noah): Less ugly name ??
```

Comment:
Any ideas of a nicer name than "attestor"? Perhaps needs to be `attestors` or `attestation` ?

Reply:
Is it all that ugly? Seems to more or less match the terminology [SPIFFE uses](https://spiffe.io/docs/latest/deploying/registering/#2-defining-the-spiffe-id-of-the-workload).

Reply:
It's fine - I think -  it just sounds "weird" to me. I think the plural "attestors" might make more sense.